app/src/main/java/ch/hikemate/app/model/profile/Profile.kt

```diff
@@ -0,0 +1,30 @@
+package ch.hikemate.app.model.profile
+
+import com.google.firebase.Timestamp
+
+/**
+ * A class representing a fitness level.
+ *
+ * @property level The level of the fitness level.
+ * @property description The description of the fitness level.
+ */
+data class FitnessLevel(
+    val level: Int,
+    val description: String,
+)
```

Comment:
I think you may be right. We should discuss it with the team on Monday.
I will leave it as is for now.

Reply:
(Discussed IRL) : As this would considerably change the PR's logic, it would represent a few hours and work and hence should be a new task in the product backlog instead of a review comment.

Reply:
Will be discussed at the stand-up meeting today and a item will be added to the backlog if relevant.